src/cloudfoundry/LogsCommandManager.ts

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright IBM Corporation 2016, 2022
+ * Copyright IBM Corporation 2016, 2023
```

Comment:
**Callout:** Updated end date to the current year. Did the same thing for the other files.